galaxygpt/AiClient.cs

```diff
@@ -75,6 +75,6 @@ The context will be given in the format of wikitext. You will be given multiple
         });
         messages.Add(new AssistantChatMessage(idk));
 
-        return messages[^1].Content[0].Text;
+        return (messages[^1].Content[0].Text, gptTokenizer.CountTokens(messages[^1].Content[0].Text));
```

Comment:
i'd prefer if we created a variable called `finalMessage` or something, then used that in place of `messages[^1].Content[0].Text`. i don't like repeating here